README.md

```diff
@@ -51,26 +91,33 @@ Cancel flux jobs
 # Cancel a job with a particular id
 $ flux cancel <jobid>
 
-# Cancel all flux jobs (with confirmation)
+# Cancel all flux jobs (with confirmation, limited to user namespace)
 $ flux cancel --all
 
 # Cancel all flux jobs (with force)
 $ flux cancel --all --force
+
+# Cancel all flux jobs (with confirmation, all users)
+$ sudo flux cancel --all --users all
```

Comment:
What if there is a user named `all`?

Reply:
To my knowledge there can't be a user named "all" that's a special token reserved in flux to tell the filter system to give back a global namespace. I came across this when I was writing the flux wrappers. So not something I'm making up now, this is already in flux. I just wanted to provide an additional example.

Reply:
Okay that works then!